doc/CMakeLists.txt

```diff
@@ -1,15 +1,9 @@
 cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
 
-project(spack-stack VERSION 1.3.1)
```

Comment:
Is there a way to get the version automatically? If we don't set the next version anywhere else then nevermind.

Reply:
We do, in `doc/src/conf.py` we can use `dev` and `develop` for the short and long version in between releases, and then I set them on the release branch. `CMakeLists.txt` unfortunately doesn't allow that, only real versions.

Would be nice if we could set the version in one place and then inherit it from there. But probably a heavy lift, given that it's only three places.